mkosi.md

```diff
@@ -315,12 +315,20 @@ a boolean argument: either "1", "yes", or "true" to enable, or "0",
 : The mirror to use for downloading the distribution packages. Expects
   a mirror URL as argument.
 
-`UseMirrorVerbatim=`, `--use-mirror-verbatim=`
+`LocalMirror=`, `--local-mirror=`
 
-: When `--mirror` is used it will be used as a plain and direct mirror instead
+: The mirror will be used as a local, plain and direct mirror instead
   of using it as a prefix for the full set of repositories normally supported
   by distributions like Fedora and CentOS. Useful for fully offline builds with
```

Comment:
How about 
```suggestion
  by distributions. Useful for fully offline builds with
```
since two sentences later we say that it is supported for all major package formats.